src/f32/funcs.rs

```diff
@@ -139,6 +197,64 @@ pub(crate) mod sse2 {
         _mm_or_ps(result, _mm_castsi128_ps(test))
     }
 
+    /// Returns a vector whose components are the corresponding components of Angles modulo 2PI.
+    #[allow(dead_code)]
+    pub(crate) unsafe fn m128_mod_angles(angles: __m128) -> __m128 {
+        // From DirectXMath: XMVectorModAngles
+        let v = _mm_mul_ps(angles, XM_RECIPROCAL_TWO_PI.m128);
+        let v = m128_round(v);
+        XM_FMADD_PS!(XM_TWO_PI.m128, v, angles)
+    }
```

Comment:
If this is no longer used it should probably be removed. Although I ask about whether this should be used below.